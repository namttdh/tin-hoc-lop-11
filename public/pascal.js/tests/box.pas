program Box;

Uses Crt;

Var I: Integer;

procedure DrawBox(X1 ,Y1, X2, Y2: Integer);
  Var I: Integer;
  begin
      GotoXY(X1 ,Y1);
      for I := X1 to X2 do write('-');
      GotoXY(X1 ,Y1+1);
      for I := Y1 + 1 to Y2 do
      begin
          GotoXY(X1,I); Write('!');
          GotoXY(X2,I); Write('!');
      end;
      GotoXY(X1 ,Y2);
      for I := X1 to X2 do Write('-');
  end; { of procedure DrawBox }

begin
    ClrScr;
    for I := 1 to 5 do DrawBox(I*4,I*2,10*I,4*I);
    DrawBox(1,1,80,23);
end.

