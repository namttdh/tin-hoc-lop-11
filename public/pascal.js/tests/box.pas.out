[2J[2;4H-------[3;4H[3;4H![3;10H![4;4H![4;10H![4;4H-------[4;8H-------------[5;8H[5;8H![5;20H![6;8H![6;20H![7;8H![7;20H![8;8H![8;20H![8;8H-------------[6;12H-------------------[7;12H[7;12H![7;30H![8;12H![8;30H![9;12H![9;30H![10;12H![10;30H![11;12H![11;30H![12;12H![12;30H![12;12H-------------------[8;16H-------------------------[9;16H[9;16H![9;40H![10;16H![10;40H![11;16H![11;40H![12;16H![12;40H![13;16H![13;40H![14;16H![14;40H![15;16H![15;40H![16;16H![16;40H![16;16H-------------------------[10;20H-------------------------------[11;20H[11;20H![11;50H![12;20H![12;50H![13;20H![13;50H![14;20H![14;50H![15;20H![15;50H![16;20H![16;50H![17;20H![17;50H![18;20H![18;50H![19;20H![19;50H![20;20H![20;50H![20;20H-------------------------------[1;1H--------------------------------------------------------------------------------[2;1H[2;1H![2;80H![3;1H![3;80H![4;1H![4;80H![5;1H![5;80H![6;1H![6;80H![7;1H![7;80H![8;1H![8;80H![9;1H![9;80H![10;1H![10;80H![11;1H![11;80H![12;1H![12;80H![13;1H![13;80H![14;1H![14;80H![15;1H![15;80H![16;1H![16;80H![17;1H![17;80H![18;1H![18;80H![19;1H![19;80H![20;1H![20;80H![21;1H![21;80H![22;1H![22;80H![23;1H![23;80H![23;1H--------------------------------------------------------------------------------